src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -215,6 +215,34 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(boolean sessionI
             tableBodyHtml.append(tableRow + Const.EOL);
         }
         
+        StringBuilder mobileHtml = new StringBuilder();
+
+        String mobilePanelTemplate = FeedbackQuestionFormTemplates.RUBRIC_SUBMISSION_FORM_MOBILE_PANEL;
+        String mobilePanelFragmentTemplate = FeedbackQuestionFormTemplates.RUBRIC_SUBMISSION_FORM_MOBILE_PANEL_FRAGMENT;
+
+        for(int j = 0 ; j < numOfRubricSubQuestions ; j++) {
```

Comment:
Yeah, we should do that in another issue, can be a <code>forFirstTimer</code> issue.